### PR TITLE
Setup viscoelastic stress build up

### PR DESCRIPTION
**Pull request by anne-glerum**
Add a script to create the setup figure for the viscoelastic stress build-up benchmark.

### CODE REVIEW

**Comment by anne-glerum** [comment]
@d-quiroga Would you know how to clean up the viscosity labels on the right graph? We don't need the two profiles here, but I'd like to know how to only plot the label of the tick I specify. Whatever I tried did not make the other labels go away.

File: viscoelastic_stress_build-up/setup_plot.py [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+ax1.get_shared_y_axes().join(ax1, ax2) # Share y axis between subplots
+plt.setp(ax1.get_xticklabels(),fontsize=12, visible=True)
+plt.setp(ax1.get_yticklabels(),fontsize=12, visible=False)
+
+# Viscosity subplot
+ax2.plot(viscosity,zs,c='k',linewidth=2)
+ax2.set_xscale('log')
+ax2.set_xlabel(r'$\eta_{\mathrm{viscous}}$ $[Pa \cdot s]$',fontsize=13)
+ax2.set_ylim(z_surface,z_bottom)
+ax2.set_xlim(5e21,5e22)
```
Comment (d-quiroga) [inline viscoelastic_stress_build-up/setup_plot.py:96]:
```suggestion
ax2.set_xlim(5e21,5e22)
plt.minorticks_off()
```
I am not sure what happens there, but this is the best solution that I could find, it deactivates the minor ticks of the log plot

Comment (anne-glerum) [reply viscoelastic_stress_build-up/setup_plot.py:96]: oh thank you, I'll give that a try!

Comment (anne-glerum) [reply viscoelastic_stress_build-up/setup_plot.py:96]: Wonderful, that worked, thanks!

**Pull request merged**